packages/lab/src/list-next/useList.ts

```diff
@@ -278,6 +278,8 @@ export const useList = ({
           onChange?.(event, { value: nextItem.dataset.value || "" });
         }
         break;
+      case "Tab":
+        break;
       case "PageDown":
       case "PageUp":
       default:
```

Comment:
I'll probably only do `event.preventDefault();` on the PageUp/Down instead of `default` instead of adding `Tab` logic